packages/plugin-antd/src/index.ts

```diff
@@ -20,6 +26,23 @@ export default (api: IApi) => {
     };
   });
 
+  const opts: IAntdOpts = api.userConfig.antd || {};
+
+  if (opts?.dark) {
+    // support dark mode, user use antd 4 by default
+    const darkThemeVars = require('antd/dist/dark-theme');
+    api.modifyDefaultConfig(config => {
+      config.theme = {
+        hack_less_umi_plugin: `true;@import "${require.resolve(
+          'antd/lib/style/color/colorPalette.less',
+        )}";`,
+        ...darkThemeVars,
+        ...(config.theme || {}),
```

Comment:
object spread 不用 fallback。

```suggestion
        ...config.theme,
```